### PR TITLE
Y24-190-3 Use API v2 TagLayout

### PR DESCRIPTION
**Pull request by sdjmchattie**
Closes #1818 

#### Changes proposed in this pull request

- Use new endpoint for TagLayout.
- Use the custom method for TagLayoutTemplate to create a TagLayout.

#### Instructions for Reviewers

_[All PRs] - Confirm PR template filled_  
_[Feature Branches] - Review code_

### CODE REVIEW

File: spec/models/labware_creators/custom_tagged_plate_spec.rb [hunk 1]
```diff
@@ -232,17 +233,18 @@ def expect_state_change_creation
           let(:tag_plate_state) { 'exhausted' }
 
           it 'creates a tagged plate' do
+            # This one will be VERY different
```
Comment (BenTopping) [inline spec/models/labware_creators/custom_tagged_plate_spec.rb:236]: Intentional comment?

File: .pryrc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+if defined?(PryByebug)
```
[the remark this answers is not in the document]
Comment (sdjmchattie) [reply .pryrc:2]: Hah.  There is a lot of debugging to do at times and I found this as a way of shortening the commands you use the most to single letters.

File: spec/models/labware_creators/custom_tagged_plate_spec.rb (hunk 1 above)
Comment (sdjmchattie) [reply spec/models/labware_creators/custom_tagged_plate_spec.rb:236]: Well it came from removed line 241 below and I'm making the same expectation, just via API V2, so hopefully it's intentional.

Comment (BenTopping) [reply spec/models/labware_creators/custom_tagged_plate_spec.rb:236]: Yeah just not sure what the difference is that the comment is suggesting.

Comment (sdjmchattie) [reply spec/models/labware_creators/custom_tagged_plate_spec.rb:236]: I don't know.  I didn't want to try to clean up existing comments and I assumed it was made for a reason.  It's very easy to put in comments that aren't useful in the future 😄

**Pull request merged**